launchcontainer/launchcontainer.py

```diff
@@ -231,18 +230,30 @@ def user_email(self):
 
         return email
 
+    @property
+    def support_url(self):
+        lcsettings = self.get_xblock_settings()
+        return lcsettings.get('support_url', DEFAULT_SUPPORT_URL)
+
+    @property
+    def timeout_secs(self):
+        lcsettings = self.get_xblock_settings()
+        return lcsettings.get('timeout_seconds', DEFAULT_LAUNCHER_TIMEOUT_SECONDS)
```

Comment:
Did you test the experience when this timeout is shorter than the timeout configured in AVL.  e.g., the timeout here is 60 seconds but the lab actually launches in 75 seconds.

Reply:
No, actually.  I'll double check but the most harm that would be done is that a "Lab taking too long to load?" message would appear, then the lab would still load.  I don't imagine we would ever configure a very short timeout or bother configuring  this at the platform level, unless for a scenario like a conference where we just wanted to bump the default 120 seconds up a little bit temporarily due to high volume.